apps/web/src/services/asset-inventory/components/MetricExplorerQuerySection.vue

```diff
@@ -45,6 +48,14 @@ const state = reactive({
         return count;
     }),
 });
+const periodText = computed<string>(() => {
+    if (isEmpty(metricExplorerPageState.period)) {
+        return '';
+    } if (metricExplorerPageGetters.isRealtimeChart) {
+        return metricExplorerPageState.period.start;
+    }
+    return `${metricExplorerPageState.period.start} ~ ${metricExplorerPageState.period.end}`;
```

Comment:
Question: is there case where `end` is missing?

Reply:
For real-time charts, since we only fetch data for **a day (or a month)**, there's no `end` parameter needed.